.gitattributes

```diff
@@ -0,0 +1,221 @@
+* text=auto eol=lf
+
+# Source code
+*.bash text eol=lf
```

Comment:
I've never personally run into problems working without a `.gitattributes`. I've looked around at a bunch of popular rust crates and can't see a `.gitattributes` file in those either.  Which leads me to believe this file is not that useful.  Can you remove it please.